apps/cpp_rtvm/tvm_runner.cc

```diff
@@ -110,6 +113,35 @@ int TVMRunner::Load(void) {
   return 0;
 }
 
+/*!
+ * \brief Specify if the run programs should be dumped to binary and reused in the next runs.
+ * \param pathToDir Path to the existed directory where pre-compiled programs should be stored.
+ */
+void TVMRunner::UsePreCompiledPrograms(std::string pathToDir) {
+  if (r_run_was_called) {
+    LOG(INFO) << "TVMRunner UsePreCompiledPrograms: should be called before first run";
+    return;
+  }
+  if (!std::filesystem::exists(pathToDir))
```

Comment:
```rtvm``` tool is not specific to Adreno or OpenCL. It works for any target. Probably we could have a target check here.

Also, Can the path be made command line arg for the tool ?

Reply:
Done